### PR TITLE
20668: Add tunnel detection time to gateway, transit and spoke

### PR DESCRIPTION
**Pull request by mlin-aviatrix**

### CODE REVIEW

File: aviatrix/resource_aviatrix_gateway.go [hunk 1]
```diff
@@ -1531,6 +1545,12 @@ func resourceAviatrixGatewayRead(d *schema.ResourceData, meta interface{}) error
 			}
 			d.Set("enable_jumbo_frame", jumboFrameStatus)
 		}
+
+		detectionTime, err := client.GetTunnelDetectionTime(gw.GwName)
```
Comment (CyrusJavan) [inline aviatrix/resource_aviatrix_gateway.go]: Need to add to list_vpcs_summary instead of using an individual API

Comment (mlin-aviatrix) [reply aviatrix/resource_aviatrix_gateway.go]: I already spoke with Cheng and he said it wasn't a good idea to put this in list_vpcs_summary.

Comment (mlin-aviatrix) [reply aviatrix/resource_aviatrix_gateway.go]: Waiting on 21203 to add tunnel detection time information to list_vpcs_summary

**Review by CyrusJavan — DISMISSED** [review]

**Review by shuoliugit — COMMENTED** [review]
Do we need to add this into data sources?

**Review by CyrusJavan — APPROVED** [review]

**Review by zhiqiangjin — APPROVED** [review]

**Pull request merged**